src/app/clusters/closure-control-server/closure-control-server.cpp

```diff
@@ -336,15 +374,65 @@ Status Instance::HandleStop(HandlerContext & ctx, const Commands::Stop::Decodabl
 Status Instance::HandleMoveTo(HandlerContext & ctx, const Commands::MoveTo::DecodableType & commandData)
 {
     MainStateEnum state = GetMainState();
-    VerifyOrReturnValue(CheckCommandStateCompatiblilty(Commands::Stop::Id, state), Status::InvalidInState);
+    VerifyOrReturnValue(CheckCommandStateCompatibility(Commands::Stop::Id, state), Status::InvalidInState);
+
+    // If all command parameters don't have a value, return InvalidCommand
+    VerifyOrReturnValue(commandData.position.HasValue() || commandData.latch.HasValue() || commandData.speed.HasValue(),
+                        Status::InvalidCommand);
+
+    if (commandData.position.HasValue())
+    {
+        VerifyOrReturnError((commandData.position.Value() != TargetPositionEnum::kUnknownEnumValue), Status::ConstraintError);
+        // If Positioning(PS) feature or not, it SHALL return a status code SUCCESS.
```

Comment:
I don't understand what "If Positioning(PS) feature or not" means.

Please fix these comments to make sense.  Probably something like:
```suggestion
        // If the Positioning(PS) feature is not supported, it SHALL return a status code SUCCESS.
```
or something, though even that is very unclear (what is "it"?) and needs a spec issue to write this stuff more clearly.  Please link to the issue here.

Reply:
Sure, will raise a spec issue.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11547